cv32e40s/tests/programs/custom/interrupt_test/interrupt_test.c

```diff
@@ -311,7 +311,7 @@ int test1() {
 // To share implementation of basic interrupt test with vector relocation tests,
 // break out the test 1 implementation here
 int test1_impl(int direct_mode) {
-    for (uint32_t i = 0; i < 32; i++) {
```

Comment:
I started debugging the interrupt test, and these changes snuck in; should not have any adverse effects (rather make the test more robust) but that test test is still broken.